### ISSUE TITLE
The portable strut does... well this...

### ISSUE THREAD

**Issue by ConCon2** [ihsoft/KAS#210]
![image](https://user-images.githubusercontent.com/10994888/29036345-b1f2539e-7b64-11e7-9bb9-1bb7e17873fc.png)
![image](https://user-images.githubusercontent.com/10994888/29036357-c05e8d4e-7b64-11e7-9a1c-d0ddd382676b.png)
![image](https://user-images.githubusercontent.com/10994888/29036376-d27dcb84-7b64-11e7-981e-8d052befe460.png)

The bug does not seem to affect the ships strength. The blocks drift around and hit the ship on there own but will move with the kerbal that linked them no matter what I do with him or were I put him. They reset back into place wile in warp but drift away when leaving warp same when loading the ship. The ship sometimes completely explodes when loading the ship. KAS is the only mod I'm running.

**Comment by ConCon2** [comment]
Half of them hit the ship the other half does not

**Comment by ihsoft** [comment]
The struts with the blocks doesn't seem to be KAS strut. I see CS-R2 parts only on the last screenshot, and they look good.

**Comment by ConCon2** [comment]
Yes the portable struts stay in place but they cause the blocks to drift away. They were all attached to the ship before I linked the portable struts together.

**Comment by ihsoft** [comment]
Could you please record a short video? And share the KSP.log file, please (the one which lives in the game's root).
